style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  text-align: center;
+  color: white;
+}
+
+#main {
+  background-color: #94c473;
+  display: flex;
+  flex-direction: column;
+  padding: 60px 80px;
+  flex: 1 1 0%;
+}
+
+#title {
+  margin-top: 50px;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  font-size: 50px;
+}
+
+.subtitle {
+  font-size: 28px;
+  margin-top: -30px;
+}
+
+#img-div {
+  display: flex;
+  background-color: white;
+  flex-direction: column;
+  padding: 10px;
+}
+
+#image {
+  height: auto;
+  max-width: 100%;
+}
+
+#img-caption {
+  font-size: 30px;
+  margin: 0 auto 0;
+  color: #94c473;
+}
+
+p {
+  margin: 30px 60px;
+  font-size: 20px;
+}
+
+p span {
+  font-weight: bold;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  max-width: 500px;
+  text-align: left;
+  margin: 0 auto 0;
+  font-size: 20px;
+}
+
+.quote {
+  width: 500px;
+  margin-top: 45px;
+  align-self: center;
+  color: cadetblue;
+}

```

Comment:
Kindly try using a different font family for the quote section,
to make it stand out ...
nice Colors combination tho!